libkbfs/folder_branch_ops.go

```diff
@@ -5711,7 +5727,7 @@ func (fbo *folderBranchOps) SyncFromServer(ctx context.Context,
 	if err := fbo.fbm.waitForDeletingBlocks(ctx); err != nil {
 		return err
 	}
-	if err := fbo.editHistory.Wait(ctx); err != nil {
+	if err := fbo.editActivity.Wait(ctx); err != nil {
```

Comment:
I know this isn't new here, but it doesn't look like we have any starvation protection. So as long as edit activity keeps coming in faster than we can process it, another client on a TLF could starve all other clients.

Reply:
True, but a) a well-behaved client with batching will only push one edit per TLF per second, and b) the starvation would only happen in `SyncFromServer`, which outside of testing I think is only used in single-op mode, which wouldn't subscribe to notifications anyway.